rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,13 +16,15 @@ toc::[]
 [id="rosa-q3-2024_{context}"]
 === Q3 2024
 
+* **{hcp-title} multi-architecture cluster update.** {hcp-title-first} clusters created before 25 July, 2024 will migrate to a multi-architecture image on their next upgrade allowing you to use {AWS} Arm-based Graviton instance types for your workloads. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-upgrade-options_rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
```

Comment:
IBMSG suggests not using future tense, so I wanted to check if you could use something besides "will" here. Maybe "should" could work?

Reply:
I think because this is a RN for a future event that will happen to clusters post release, it is appropriate here and we should keep it.